views/js/qtiCreator/widgets/interactions/helpers/answerState.js

```diff
@@ -236,7 +246,8 @@ define([
                 template : template,
                 templates : _getAvailableRpTemplates(interaction, options.rpTemplates, widget.options.allowCustomTemplate),
                 listOfBaseType : listOfBaseType,
-                listOfBaseTypes : _getAvailableListOfBaseTypes(),
+                listOfBaseTypes : _getAvailableListOfBaseTypes(listOfBaseType),
+                textEntryInteraction: interaction.qtiClass === 'textEntryInteraction',
```

Comment:
I don't like such hardcoded names of interaction. Maybe you can load it from `QtiElements.getAvailableAuthoringElements`? https://github.com/oat-sa/extension-tao-itemqti/blob/develop/views/js/qtiCreator/helper/qtiElements.js#L300

Reply:
Take a look now ;)